web/template/watch.gohtml

```diff
@@ -424,6 +424,22 @@
             </div>
         {{end}}
 
+        <!-- Transcript -->
+        <div id="transcript-desktop" x-cloak="" x-show="sidebar === watch.SidebarState.Transcript && transcriptAvailable"
```

Comment:
```suggestion
        <div id="transcript-desktop" x-cloak x-show="sidebar === watch.SidebarState.Transcript && transcriptAvailable"
```

Reply:
Looks like a copy-paste error like the one related to the unnecessary `'` :sweat_smile:  
![image](https://github.com/user-attachments/assets/b28d30b6-a0c9-4b1f-a27a-0e09f998300e)

I fixed it in my latest commit.